src/components/overview.js

```diff
@@ -149,6 +149,9 @@ var OverviewBlur = class OverviewBlur {
         Main.layoutManager.overviewGroup.get_children().forEach(actor => {
             if (actor.constructor.name === 'Meta_BackgroundActor') {
```

Comment:
Not sure I know the code base, but are there different names other than 'Meta_BackgroundActor' ? 

Maybe that causes the leak.